.travis.yml

```diff
@@ -21,7 +21,16 @@ matrix:
 
 branches:
   only:
+    # master branch's j2objc-gradle submodule should track
+    # only the latest production release of j2objc-gradle.
+    # For now it tracks a particular unreleased commit because v0.5.0-alpha
+    # must be released for depedency features to work.
+    # Maintainers must manually update submodule to the latest release.
     - master
+    # pluginHead branch is used to always test the latest
+    # HEAD of the j2objc-gradle repository.  They are automatically
```

Comment:
I think we should standardize on a single space after a period. In the plugin we have 58 cases of '. ' (single space) in comments and 17 cases of '.  ' (double space). I think single space is far more common.